src/pages/profile/edit/index.tsx

```diff
@@ -24,6 +27,13 @@ const Edit = () => {
   const [isChanged, setIsChanged] = useState(false);
   const [profileImage, setProfileImage] = useState(initialInfos.profileImage);
 
+  useEffect(() => {
+    if (location.state?.croppedImage) {
+      setProfileImage(location.state.croppedImage);
+      setIsChanged(true);
+    }
+  }, [location.state?.croppedImage]);
+
```

Comment:
```suggestion
  useEffect(() => {
    const croppedImage = location.state?.croppedImage;
    if (croppedImage) {
      setProfileImage(croppedImage);
      setIsChanged(true);
    }
  }, [location.state?.croppedImage]);

```

가독성 향상을 위해 이렇게 해보는건 어떨까요